readme.md

```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [React Hooks](https://github.com/glauberfc/awesome-react-hooks#readme) - A new feature that lets you use state and other React features without writing a class.
```

Comment:
Should probably come before Relay, if you are adding this in alphabetical order.

Reply:
Hmm...Are this alphabetical order required @sindresorhus

Reply:
If you have to ask, you have clearly not read the PR guidelines:

> By submitting this pull request I confirm I've read and complied with the below requirements 🖖